FlashDevelop/MainForm.cs

```diff
@@ -1700,6 +1701,9 @@ protected override bool ProcessCmdKey(ref Message msg, Keys keyData)
              */
             else if (ShortcutKeysManager.IsValidExtendedShortcutFirst(keyData) && !appSettings.DisableExtendedShortcutKeys)
             {
+                if (ShortcutManager.AllShortcuts.FirstOrDefault(s => s.First == keyData) == ShortcutKeys.None)
+                    return false;
+
```

Comment:
If you do it that way it would be still wrong, because it would say Alt+T is undefined, return true, and the mnemonic wouldn't be executed. I don't see in VS notifyications about undefined shortcuts for simple cases, I'd say if you want to have that one then you'd have to check all the menustrips and controls to check for mnemonics (as a button, label, etc may have a mnemonic).